ql/src/java/org/apache/hadoop/hive/ql/exec/repl/incremental/IncrementalLoadTasksBuilder.java

```diff
@@ -164,6 +165,12 @@ public IncrementalLoadTasksBuilder(String dbName, String tableName, String loadP
                   lastEventid);
         }
       }
+
+      ReplSetFirstIncLoadFlagDesc desc = new ReplSetFirstIncLoadFlagDesc(dbName, tableName);
```

Comment:
Can we combine this with update last replID alterDb task?

Reply:
new ddl task is created to make it simpler with table level and warehouse level support

Reply:
OK